buildroot-external/rootfs-overlay/etc/sysctl.d/30-cp15barrier.conf

```diff
@@ -0,0 +1,3 @@
+# disable "deprecated CP15 Barrier instruction" warnings
+# https://www.kernel.org/doc/Documentation/arm64/legacy_instructions.txt
+abi.cp15_barrier = 2
```

Comment:
Well, not within the Buildroot/Linux OS itself, but e.g. the RaspberryMatic HA add-on comes with 32bit binaries (because the upstream vendor (eQ3) didn't yet supply 64bit binaries) and thus with a complete 32bit multilib environment (see https://github.com/jens-maus/RaspberryMatic/tree/master/buildroot-external/package/multilib32). And thus, 32bit apps actually (can) run within a docker environment and thus uses the CP15 barrier instructions. Thus, I would prefer to keep that minor sysctl tuning because it stops the kernel flooding for which I didn't find any other way to silence these messages.